### ISSUE TITLE
Deep explainer when the model output is logit

### ISSUE THREAD

**Issue by PayamDiba** [shap/shap#2416]
I was wondering if the functionality of the deepSHAP explainer changes when the TensorFlow model outputs logits rather than class probabilities (for classification task)?
